### DISCUSSION TITLE
how to modify ordinate value in  population simulation?

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#831] [Questions & Problems] [answered]
Dear to all,
I want to modify the maximum and minimum of the ordinate value in  population simulation, how to do it?
![image](https://user-images.githubusercontent.com/63133525/89908424-70ef0c00-dc20-11ea-90e4-9c337ce25028.png)

**Comment by tobiasK2001** [comment, marked as answer]
Hi,
unfortunately this is not possible by directly putting in numbers in PK-sim. You can just use the clumsy zoom in or reset zoom function. Please make a feature request for this.
Until this is solved you might try this humble workaround:
Insert fake observed data values to trim the ordinate like e.f 20000 ng/ml at time = 0 and last timepoint measured. Drag and drop the data in you plot and change colour to white, so that are not disturbing so much anymore (so want will not get invisible though). The PK-sim auto zoom function will try to adapt zoom around this data. You might try different values for optimal results.

Hope that helps.

Best, Tobias

**Comment by arketkar** [comment]
Hi @tobiasK2001 

As suggested by you I think that would be a good feature request, and it would be very handy for users. Thanks.

Sorry however I do not know how to raise a feature request. @Wenjing1018 might do that if he knows. 

Regards.
Anant

**Comment by Wenjing1018** [comment]
@tobiasK2001 
 Your reply is very helpful to me，I also want to know how to get rid of the background squares above？

**Comment by tobiasK2001** [comment]
I haven't found a way to do this, Sorry! Please make a feature request for that as well.

Best, Tobais

**Comment by Wenjing1018** [comment]
Dear tobiasK2001,
 Thank you for your reply,but I don't know how to make a feature request.
Yours Sincerely!

**Comment by prvmalik** [comment]
You may export the plot to a .csv file and upload it into other graphing softwares, in the meantime. I prefer MATLAB.

**Comment by tobiasK2001** [comment]
I created  Issues/ requests here:[https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1546](url)

and here[https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1547](url)
please give a positive reaction to them.

Best, Tobias

**Comment by arketkar** [comment]
> I created Issues/ requests here:[https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1546](url)
> 
> and here[https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1547](url)
> please give a positive reaction to them.
> 
> Best, Tobias

Hi @tobiasK2001 

Sorry I am not able to see anything after clicking on both the links shared by you.

Regards.
Anant

**Comment by tobiasK2001** [comment]
hmm, try this ones:

[https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1547#]

[https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1546#]
